control-plane/subcommand/install-cni/testdata/ZZZ-consul-cni-kubeconfig.golden

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+clusters:
+- cluster:
+    certificate-authority: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0
+    server: https://172.30.0.1:443
```

Comment:
so should this have https://[172.30.0.1]:443?

Reply:
I think it works either way. I have added a todo to double check when I am writing acceptance tests.

Reply:
If other folks are using `[]` then I think safer to keep it like that.

Reply:
Done. Added to the tests and code.